docs/spectrum/fitting.rst

```diff
@@ -17,75 +17,73 @@ Getting Started
 ===============
 
 The following example shows how to fit a power law simultaneously to two
-simulated crab runs using the `~gammapy.spectrum.SpectrumFit` class.
+simulated crab runs using the `~gammapy.utils.fitting.Fit` class.
 
 .. code-block:: python
 
-    from gammapy.spectrum import SpectrumObservation, SpectrumObservationList, SpectrumFit
+    from gammapy.spectrum import SpectrumDatasetOnOff
+    from gammapy.utils.fitting import Fit
     from gammapy.spectrum.models import PowerLaw
     import matplotlib.pyplot as plt
 
     path = "$GAMMAPY_DATA/joint-crab/spectra/hess/"
-    obs1 = SpectrumObservation.read(path + "pha_obs23523.fits")
-    obs2 = SpectrumObservation.read(path + "pha_obs23592.fits")
-    obs_list = SpectrumObservationList([obs1, obs2])
+    obs_1 = SpectrumDatasetOnOff.read(path + "pha_obs23523.fits")
+    obs_2 = SpectrumDatasetOnOff.read(path + "pha_obs23592.fits")
 
     model = PowerLaw(
         index=2,
         amplitude='1e-12  cm-2 s-1 TeV-1',
         reference='1 TeV',
     )
 
-    fit = SpectrumFit(obs_list=obs_list, model=model, stat='wstat')
+    obs_1.model = model
+    obs_2.model = model
+
+    fit = Fit([obs_1, obs_2])
     result = fit.run()
 
-You can check the fit results by looking at
-`~gammapy.spectrum.SpectrumFitResult` that is attached to the
-`~gammapy.spectrum.SpectrumFit` for each observation.
+model.parameters.covariance = result.parameters.covariance
+You can check the fit results by looking at the result and model object:
 
 .. code-block:: python
 
     >>> print(result)
-    FitResult
+
+        OptimizeResult
 
         backend    : minuit
         method     : minuit
         success    : True
-        nfev       : 99
-        total stat : 63.00
+        nfev       : 115
```

Comment:
I was also surprised by this change, because in the tests not a single number changed. Interestingly the sherpa example in the docs changed as well and also gave different, but consistent numbers with the other example. So I think we probably forgot to update this docs example to the latest version of the `hess-dl3` dataset. The change is definitely not related to the `SpectrumFit` -> `Fit` transition.